tests/test_fillnodata.py

```diff
@@ -1,37 +1,33 @@
 import logging
 import sys
+
 import numpy as np
 import pytest
 
 import rasterio
 from rasterio.fill import fillnodata
 
 
-logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
+@pytest.fixture(scope='session')
+def hole_in_ones():
```

Comment:
@sgillies I hate to sound like a critic - because this is clever - but I'm concerned that the name doesn't make clear what it is.  Something like `basic_image_with_hole` while much more boring, makes it a little more clear what it is.  I think of our fixtures as building blocks; ideally we'd use them lots of places consistently.  Obvious names help reinforce that.